tests/providers/test_soniox_backend.py

```
@@ -36,6 +38,49 @@ def _make_session(
     )
 
 
+@pytest.mark.asyncio
+async def test_soniox_backend_open_cancellation_closes_started_session(monkeypatch) -> None:
+    started = asyncio.Event()
+    closed = asyncio.Event()
+    sessions = []
+
+    class PartialSession:
+        def __init__(self, **_kwargs) -> None:
```

Ruff (0.16.1)

[warning] Missing type annotation for `**_kwargs`

(ANN003)